Main.java

```diff
@@ -0,0 +1,13 @@
+class SimpleCommandLine
```

Comment:
File name and class name should match (e.g. the file "Main.java" should have the class name "Main").